packages/cli/src/commands/doctor/healthchecks/__tests__/androidSDK.test.ts

```diff
@@ -34,7 +34,7 @@ describe('androidSDK', () => {
   let environmentInfo: EnvironmentInfo;
 
   beforeAll(async () => {
-    environmentInfo = await getEnvironmentInfo();
+    environmentInfo = (await getEnvironmentInfo()) as EnvironmentInfo;
```

Comment:
This is not good. Let's type `getEnvironmentInfo` properly so it returns correct type based on the arguments